fixtures/secrets/variables.tf

```diff
@@ -0,0 +1,4 @@
+variable "license_file" {
+  type        = string
```

Comment:
I think we may want to keep the variables consistently typed across the three clouds' modules. Here in the Azure module, the license file variable is an [object](https://github.com/hashicorp/terraform-azurerm-terraform-enterprise/blob/main/fixtures/secrets/variables.tf#L12-L15).

Reply:
It is used as a string in GCP, for consistency I have converted to object

Reply:
never mind, I read it wrong

Reply:
This still needs to be addressed 🥺